e2e-test/e2e.sh

```diff
@@ -24,8 +24,19 @@ function setup() {
   docker build -t us-docker.pkg.dev/fhir-proxy-build/stable/fhir-access-proxy:${BUILD_ID} .
   docker-compose -f docker/keycloak/config-compose.yaml \
                  up --force-recreate --remove-orphans -d --quiet-pull
+  # TODO find a way to expose docker container logs in the output; currently
```

Comment:
I think we need to add a logging label in the compose files to direct logs elsewhere: 

https://docs.docker.com/config/containers/logging/configure/

for example, we can redirect to journald and print them when exiting the script:
https://stackoverflow.com/questions/43663013/access-logs-of-a-killed-docker-container

Reply:
Yes, agreed; for now I just leave this TODO.